pcl_conversions/include/pcl_conversions/pcl_conversions.h

```diff
@@ -717,191 +724,193 @@ namespace pcl {
 
 } // namespace pcl
 
-namespace ros
+namespace message_filters
```

Comment:
Why did you change the namespace?

Reply:
I think the templates for DefaultMessageCreator is in the message_filter namespace in ROS2; I get this error message trying to use the `ros` namespace 
```
-- stderr: pcl_conversions
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/test/test_pcl_conversions.cpp:5:
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/include/pcl_conversions/pcl_conversions.h:72:
In file included from /usr/local/include/pcl-1.9/pcl/io/pcd_io.h:791:
In file included from /usr/local/include/pcl-1.9/pcl/io/impl/pcd_io.hpp:49:
/usr/local/include/pcl-1.9/pcl/io/low_level_io.h:154:85: warning: missing field 'fst_bytesalloc' initializer [-Wmissing-field-initializers]
      ::fstore_t store = {F_ALLOCATEALL | F_ALLOCATECONTIG, F_PEOFPOSMODE, 0, length};
                                                                                    ^
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/test/test_pcl_conversions.cpp:5:
/Users/andreasklintberg/personal/perception_pcl/pcl_conversions/include/pcl_conversions/pcl_conversions.h:730:10: error: no template named 'DefaultMessageCreator'; did you mean 'message_filters::DefaultMessageCreator'?
  struct DefaultMessageCreator<pcl::PCLPointCloud2>
         ^~~~~~~~~~~~~~~~~~~~~
         message_filters::DefaultMessageCreator
/Users/andreasklintberg/personal/ros2/ros2-osx/include/message_filters/message_event.h:54:8: note: 'message_filters::DefaultMessageCreator' declared here
struct DefaultMessageCreator
       ^
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/test/test_pcl_conversions.cpp:5:
/Users/andreasklintberg/personal/perception_pcl/pcl_conversions/include/pcl_conversions/pcl_conversions.h:730:10: error: class template specialization of 'DefaultMessageCreator' not in a namespace enclosing 'message_filters'
  struct DefaultMessageCreator<pcl::PCLPointCloud2>
         ^
/Users/andreasklintberg/personal/ros2/ros2-osx/include/message_filters/message_event.h:54:8: note: explicitly specialized declaration is here
struct DefaultMessageCreator
       ^
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/test/test_pcl_conversions.cpp:5:
/Users/andreasklintberg/personal/perception_pcl/pcl_conversions/include/pcl_conversions/pcl_conversions.h:770:23: error: no template named 'HasHeader'; did you mean 'message_filters::message_traits::HasHeader'?
    template<> struct HasHeader<pcl::PCLPointCloud2> : std::true_type {};
                      ^~~~~~~~~
                      message_filters::message_traits::HasHeader
/Users/andreasklintberg/personal/ros2/ros2-osx/include/message_filters/message_traits.h:46:46: note: 'message_filters::message_traits::HasHeader' declared here
template<typename M, typename = void> struct HasHeader : public std::false_type {};
                                             ^
In file included from /Users/andreasklintberg/personal/perception_pcl/pcl_conversions/test/test_pcl_conversions.cpp:5:
/Users/andreasklintberg/personal/perception_pcl/pcl_conversions/include/pcl_conversions/pcl_conversions.h:770:23: error: class template specialization of 'HasHeader' not in a namespace enclosing 'message_traits'
    template<> struct HasHeader<pcl::PCLPointCloud2> : std::true_type {};
                      ^
/Users/andreasklintberg/personal/ros2/ros2-osx/include/message_filters/message_traits.h:46:46: note: explicitly specialized declaration is here
template<typename M, typename = void> struct HasHeader : public std::false_type {};
```

Reply:
if all this stuff is commented out, why leave the 2-3 things in it + namespace? This isn't a critique, I just don't know, is there something I dont know that needs that stuff?